samples/bot-formatting-cards/csharp/BotFormattingCards/Resources/adaptivePeoplePersonaCardIcon.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "$schema": "https://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.0.0",
```

Comment:
Please confirm with engineering team whether we need to use version 1.0.0 or the latest one.

Reply:
Fixed